docs/maven-setting.md

```diff
@@ -138,3 +138,29 @@ And you will need to add following repositories to get above dependencies resolv
 </repositories>
 {% endhighlight %}
 {% endif %}
+
+For SBT the configuration above, adding the two external modules as well, is equivalent to the following:
+
+```
+resolvers ++= Seq(
+  "Sonatype releases" at "http://oss.sonatype.org/content/repositories/releases/",
+  "Patrik at Bintray" at "http://dl.bintray.com/patriknw/maven",
+  "Cloudera repo" at "https://repository.cloudera.com/artifactory/cloudera-repos",
+  "vincent" at "http://dl.bintray.com/fvunicorn/maven",
+  "non" at "http://dl.bintray.com/non/maven",
+  "Maven repo" at "http://repo.maven.apache.org/maven2",
+  "Maven1 repo" at "http://repo1.maven.org/maven2",
+  "Maven2 repo" at "http://mvnrepository.com/artifact"
+)
+
+val gearpumpVersion = "0.7.5"
```

Comment:
Can you replace "0.7.5" with template string "{{ site.GEARPUMP_VERSION }}"

It will be auto-expanded when generating documents.